[2025-01-26T00:38:24.037-0300] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-01-26T00:38:24.044-0300] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: InsertIntoElastic.inject_data_into_es scheduled__2021-01-20T00:00:00+00:00 [queued]>
[2025-01-26T00:38:24.048-0300] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: InsertIntoElastic.inject_data_into_es scheduled__2021-01-20T00:00:00+00:00 [queued]>
[2025-01-26T00:38:24.048-0300] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2025-01-26T00:38:24.056-0300] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): inject_data_into_es> on 2021-01-20 00:00:00+00:00
[2025-01-26T00:38:24.062-0300] {standard_task_runner.py:72} INFO - Started process 111468 to run task
[2025-01-26T00:38:24.064-0300] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'InsertIntoElastic', 'inject_data_into_es', 'scheduled__2021-01-20T00:00:00+00:00', '--job-id', '4491', '--raw', '--subdir', 'DAGS_FOLDER/DAGInjectElastic.py', '--cfg-path', '/tmp/tmp4uu9jnrv']
[2025-01-26T00:38:24.065-0300] {standard_task_runner.py:105} INFO - Job 4491: Subtask inject_data_into_es
[2025-01-26T00:38:24.088-0300] {task_command.py:467} INFO - Running <TaskInstance: InsertIntoElastic.inject_data_into_es scheduled__2021-01-20T00:00:00+00:00 [running]> on host lucas-Aspire-A514-54
[2025-01-26T00:38:24.133-0300] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='lucas' AIRFLOW_CTX_DAG_ID='InsertIntoElastic' AIRFLOW_CTX_TASK_ID='inject_data_into_es' AIRFLOW_CTX_EXECUTION_DATE='2021-01-20T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-01-20T00:00:00+00:00'
[2025-01-26T00:38:24.133-0300] {taskinstance.py:731} INFO - ::endgroup::
[2025-01-26T00:38:24.252-0300] {taskinstance.py:3311} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/lucas/airflow_env/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 767, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/lucas/airflow_env/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 733, in _execute_callable
    return ExecutionCallableRunner(
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/lucas/airflow_env/lib/python3.12/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/lucas/airflow_env/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 422, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/lucas/airflow_env/lib/python3.12/site-packages/airflow/operators/python.py", line 238, in execute
    return_value = self.execute_callable()
                   ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/lucas/airflow_env/lib/python3.12/site-packages/airflow/operators/python.py", line 256, in execute_callable
    return runner.run(*self.op_args, **self.op_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/lucas/airflow_env/lib/python3.12/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/lucas/airflow/dags/DAGInjectElastic.py", line 71, in inject_data_into_es
    df_json=df.to_dict('pactrat')
            ^^^^^^^^^^^^^^^^^^^^^
  File "/home/lucas/airflow_env/lib/python3.12/site-packages/pandas/util/_decorators.py", line 333, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/lucas/airflow_env/lib/python3.12/site-packages/pandas/core/frame.py", line 2178, in to_dict
    return to_dict(self, orient, into=into, index=index)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/lucas/airflow_env/lib/python3.12/site-packages/pandas/core/methods/to_dict.py", line 272, in to_dict
    raise ValueError(f"orient '{orient}' not understood")
ValueError: orient 'pactrat' not understood
[2025-01-26T00:38:24.259-0300] {taskinstance.py:1225} INFO - Marking task as UP_FOR_RETRY. dag_id=InsertIntoElastic, task_id=inject_data_into_es, run_id=scheduled__2021-01-20T00:00:00+00:00, execution_date=20210120T000000, start_date=20250126T033824, end_date=20250126T033824
[2025-01-26T00:38:24.282-0300] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-01-26T00:38:24.282-0300] {standard_task_runner.py:124} ERROR - Failed to execute job 4491 for task inject_data_into_es (orient 'pactrat' not understood; 111468)
Traceback (most recent call last):
  File "/home/lucas/airflow_env/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py", line 117, in _start_by_fork
    ret = args.func(args, dag=self.dag)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/lucas/airflow_env/lib/python3.12/site-packages/airflow/cli/cli_config.py", line 49, in command
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/lucas/airflow_env/lib/python3.12/site-packages/airflow/utils/cli.py", line 116, in wrapper
    return f(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^
  File "/home/lucas/airflow_env/lib/python3.12/site-packages/airflow/cli/commands/task_command.py", line 483, in task_run
    task_return_code = _run_task_by_selected_method(args, _dag, ti)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/lucas/airflow_env/lib/python3.12/site-packages/airflow/cli/commands/task_command.py", line 256, in _run_task_by_selected_method
    return _run_raw_task(args, ti)
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/lucas/airflow_env/lib/python3.12/site-packages/airflow/cli/commands/task_command.py", line 341, in _run_raw_task
    return ti._run_raw_task(
           ^^^^^^^^^^^^^^^^^
  File "/home/lucas/airflow_env/lib/python3.12/site-packages/airflow/utils/session.py", line 97, in wrapper
    return func(*args, session=session, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/lucas/airflow_env/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 3005, in _run_raw_task
    return _run_raw_task(
           ^^^^^^^^^^^^^^
  File "/home/lucas/airflow_env/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 273, in _run_raw_task
    TaskInstance._execute_task_with_callbacks(
  File "/home/lucas/airflow_env/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 3159, in _execute_task_with_callbacks
    result = self._execute_task(context, task_orig)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/lucas/airflow_env/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 3183, in _execute_task
    return _execute_task(self, context, task_orig)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/lucas/airflow_env/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 767, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/lucas/airflow_env/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 733, in _execute_callable
    return ExecutionCallableRunner(
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/lucas/airflow_env/lib/python3.12/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/lucas/airflow_env/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 422, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/lucas/airflow_env/lib/python3.12/site-packages/airflow/operators/python.py", line 238, in execute
    return_value = self.execute_callable()
                   ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/lucas/airflow_env/lib/python3.12/site-packages/airflow/operators/python.py", line 256, in execute_callable
    return runner.run(*self.op_args, **self.op_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/lucas/airflow_env/lib/python3.12/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/lucas/airflow/dags/DAGInjectElastic.py", line 71, in inject_data_into_es
    df_json=df.to_dict('pactrat')
            ^^^^^^^^^^^^^^^^^^^^^
  File "/home/lucas/airflow_env/lib/python3.12/site-packages/pandas/util/_decorators.py", line 333, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/lucas/airflow_env/lib/python3.12/site-packages/pandas/core/frame.py", line 2178, in to_dict
    return to_dict(self, orient, into=into, index=index)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/lucas/airflow_env/lib/python3.12/site-packages/pandas/core/methods/to_dict.py", line 272, in to_dict
    raise ValueError(f"orient '{orient}' not understood")
ValueError: orient 'pactrat' not understood
[2025-01-26T00:38:24.317-0300] {local_task_job_runner.py:266} INFO - Task exited with return code 1
[2025-01-26T00:38:24.335-0300] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-01-26T00:38:24.335-0300] {local_task_job_runner.py:245} INFO - ::endgroup::
